extensions/multitenancy.md

```diff
@@ -0,0 +1,138 @@
+#Multitenancy Extension
```

Comment:
This page should be part of the 4.6 documentation, since that's when we introduced it.
For that to happen, you need to point the pull request to the `4.6` branch, instead of `master`.

Note that adjusting this just may require you to recreate this pull request.